packages/provider/src/tasks/tasks.ts

```diff
@@ -576,9 +576,10 @@ export class Tasks {
     }
 
     async saveCaptchaEvent(events: StoredEvents, accountId: string) {
-        if (!this.config.mongoAtlasUri) {
+        if (!this.config.devOnlyWatchEvents || !this.config.mongoEventsUri) {
+            console.log('Dev watch events not set to true, not saving events')
             return
         }
-        await saveCaptchaEvent(events, accountId, this.config.mongoAtlasUri)
+        await saveCaptchaEvent(events, accountId, this.config.mongoEventsUri)
```

Comment:
are we changing the place we submit captchas too also?

Reply:
This is just terminology change for clarity

Reply:
okay cool, no prob then :)